### PR TITLE
MINOR: fix docs upgrade instructions fro 3.9 and 3.8

### PR DESCRIPTION
**Pull request by jlprat**
We forgot to add the upgrade instructions when preparing for the releases.

*More detailed description of your change,
if necessary. The PR title and PR message become
the squashed commit message, so use a separate
comment to ping reviewers.*

*Summary of testing strategy (including rationale)
for the feature or bug fix. Unit and/or integration
tests are expected for any behaviour change and
system tests should be considered for larger changes.*

### Committer Checklist (excluded from commit message)
- [ ] Verify design and implementation 
- [ ] Verify test coverage and CI build status
- [ ] Verify documentation (including upgrade notes)

### CODE REVIEW

**Comment by jlprat** [comment]
ping @cmccabe I updated the `kafka-site` page, but we need to keep 3.9 up to date.

**Review by chia7712 — APPROVED** [review]

**Comment by jlprat** [comment]
As I don't know if @cmccabe has already started to build RC1, I leave it to him to decide if we merge it now or later.

File: docs/upgrade.html [hunk 1]
```diff
@@ -20,6 +20,68 @@
...
+        <li>Update server.properties on all brokers and add the following properties. CURRENT_KAFKA_VERSION refers to the version you
+            are upgrading from. CURRENT_MESSAGE_FORMAT_VERSION refers to the message format version currently in use. If you have previously
+            overridden the message format version, you should keep its current value. Alternatively, if you are upgrading from a version prior
+            to 0.11.0.x, then CURRENT_MESSAGE_FORMAT_VERSION should be set to match CURRENT_KAFKA_VERSION.
+            <ul>
+                <li>inter.broker.protocol.version=CURRENT_KAFKA_VERSION (e.g. <code>3.8</code>, <code>3.7</code>, etc.)</li>
+                <li>log.message.format.version=CURRENT_MESSAGE_FORMAT_VERSION  (See <a href="#upgrade_10_performance_impact">potential performance impact
+                    following the upgrade</a> for the details on what this configuration does.)</li>
+            </ul>
+            If you are upgrading from version 0.11.0.x or above, and you have not overridden the message format, then you only need to override
```
Comment (cmccabe) [inline docs/upgrade.html]: We don't support upgrading from Kafka 0.11.0 any more. The oldest version we can safely upgrade from is Kafka 2.4.0. We should also specify that ZooKeeper should be upgraded to 3.8.3 prior to the upgrade.

File: docs/upgrade.html [hunk 2]
```diff
@@ -45,6 +107,67 @@ <h5><a id="upgrade_390_notable" href="#upgrade_390_notable">Notable changes in 3
...
+        <li>Update server.properties on all brokers and add the following properties. CURRENT_KAFKA_VERSION refers to the version you
+            are upgrading from. CURRENT_MESSAGE_FORMAT_VERSION refers to the message format version currently in use. If you have previously
+            overridden the message format version, you should keep its current value. Alternatively, if you are upgrading from a version prior
+            to 0.11.0.x, then CURRENT_MESSAGE_FORMAT_VERSION should be set to match CURRENT_KAFKA_VERSION.
+            <ul>
+                <li>inter.broker.protocol.version=CURRENT_KAFKA_VERSION (e.g. <code>3.7</code>, <code>3.6</code>, etc.)</li>
+                <li>log.message.format.version=CURRENT_MESSAGE_FORMAT_VERSION  (See <a href="#upgrade_10_performance_impact">potential performance impact
+                    following the upgrade</a> for the details on what this configuration does.)</li>
+            </ul>
+            If you are upgrading from version 0.11.0.x or above, and you have not overridden the message format, then you only need to override
```
Comment (cmccabe) [inline docs/upgrade.html]: We don't support upgrading from Kafka 0.11.0 any more. The oldest version we can safely upgrade from is Kafka 2.4.0. We should also specify that ZooKeeper should be upgraded to 3.8.3 prior to the upgrade.

File: docs/upgrade.html [hunk 3]
```diff
@@ -45,6 +107,67 @@ <h5><a id="upgrade_390_notable" href="#upgrade_390_notable">Notable changes in 3
...
+    <ol>
+        <li>Upgrade the brokers one at a time: shut down the broker, update the code, and restart it. Once you have done so, the
+            brokers will be running the latest version and you can verify that the cluster's behavior and performance meets expectations.
+        </li>
+        <li>Once the cluster's behavior and performance has been verified, bump the metadata.version by running
+            <code>
+                bin/kafka-features.sh upgrade --metadata 3.8
+            </code>
+        </li>
+        <li>Note that cluster metadata downgrade is not supported in this version since it has metadata changes.
```
Comment (cmccabe) [inline docs/upgrade.html]: It's sufficient to say "Note that cluster metadata downgrade is not supported in this software version."

File: docs/upgrade.html [hunk 4]
```diff
@@ -20,6 +20,68 @@
...
+    <ol>
+        <li>Upgrade the brokers one at a time: shut down the broker, update the code, and restart it. Once you have done so, the
+            brokers will be running the latest version and you can verify that the cluster's behavior and performance meets expectations.
+        </li>
+        <li>Once the cluster's behavior and performance has been verified, bump the metadata.version by running
+            <code>
+                bin/kafka-features.sh upgrade --metadata 3.9
+            </code>
+        </li>
+        <li>Note that cluster metadata downgrade is not supported in this version since it has metadata changes.
```
Comment (cmccabe) [inline docs/upgrade.html]: It's sufficient to say "Note that cluster metadata downgrade is not supported in this software version."

**Comment by jlprat** [comment]
Thanks for the feedback @cmccabe. I updated the PR.

**Pull request merged**